spec/requests/items_spec.rb

```diff
@@ -53,6 +53,12 @@
   describe "GET /edit" do
     it "renders a successful response" do
       item = Item.create! valid_attributes
+      user = FactoryBot.create(:user, password: "password")
```

Comment:
Do we have to specify the password here?

Reply:
Das weiß ich nicht genau, ich habe den Test so gelassen und nur die Berechtigung für den User hinzugefügt, damit er ein Item löschen kann.